libs/wordpress-plugin/plugin/trunk/server/block-db-table.php

```diff
@@ -65,7 +83,7 @@ function block_protocol_migrate()
 
   if ($saved_version < 2) {
     block_protocol_migration_1();
-    update_site_option('block_protocol_db_migration_version', 2);
+    block_protocol_set_migration_version_to(2);
```

Comment:
We previously set the migration version regardless of success. We now check if there's a DB failure before setting the number.